package.json

```diff
@@ -46,6 +46,12 @@
     "babel-preset-react-native": "^1.4.0",
     "browserify": "^13.0.0",
     "react": "^15.4.0",
-    "react-dom": "^15.4.0"
+    "react-dom": "^15.4.0",
```

Comment:
Should this be 16 now?

Reply:
we could, but then the interactive docs will need some refactoring (they use createClass right now). There's nothing about animated that's specific to 15 or 16, so it doesn't need to be changed, but we should move to 16 soon. I want to spend some time integrating animated w/ fiber's scheduling, and so at that time i'll definitely up the dev dep. don't think it should be in this PR though.

Reply:
Ah gotcha, I'll refactor those and make them not just random inline code